core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -57,28 +68,73 @@ public class EnrolleeImportService {
         this.registrationService = registrationService;
         this.enrollmentService = enrollmentService;
         this.profileService = profileService;
         this.enrolleeExportService = enrolleeExportService;
         this.surveyResponseService = surveyResponseService;
         this.participantTaskService = participantTaskService;
         this.portalService = portalService;
+        this.importService = importService;
+        this.importItemService = importItemService;
     }
 
     @Transactional
     /**
      * imports the enrollees serialized in the inputstream to the given environment
      */
-    public void importEnrollees(String portalShortcode, String studyShortcode, StudyEnvironment studyEnv, InputStream in) {
+    public Import importEnrollees(String portalShortcode, String studyShortcode, StudyEnvironment studyEnv, InputStream in, UUID adminId) {
+        Import dataImport = Import.builder()
+                .responsibleUserId(adminId)
+                .studyEnvironmentId(studyEnv.getId())
+                .responsibleUserId(adminId)
+                .importType(ImportType.PARTICIPANT)
+                .status(ImportStatus.PROCESSING)
+                .createdAt(Instant.now())
+                .lastUpdatedAt(Instant.now())
+                .build();
+        dataImport = importService.create(dataImport);
+        log.info("Started Import ID: {}", dataImport.getId());
         List<Map<String, String>> enrolleeMaps = generateImportMaps(in);
         for (Map<String, String> enrolleeMap : enrolleeMaps) {
-            Enrollee enrollee = importEnrollee(portalShortcode, studyShortcode, studyEnv, enrolleeMap, IMPORT_OPTIONS);
+            Enrollee enrollee = null;
+            ImportItem importItem;
+            try {
+                enrollee = importEnrollee(portalShortcode, studyShortcode, studyEnv, enrolleeMap, IMPORT_OPTIONS);
+                importItem = ImportItem.builder()
+                        .createdEnrolleeId(enrollee.getId())
+                        .importId(dataImport.getId())
+                        .createdParticipantUserId(enrollee.getParticipantUserId())
+                        .createdAt(Instant.now())
+                        .lastUpdatedAt(Instant.now())
+                        .status(ImportItemStatus.SUCCESS).build();
+            } catch (Exception e) {
+                importItem = ImportItem.builder()
+                        .createdEnrolleeId(enrollee.getId())
+                        .importId(dataImport.getId())
+                        .createdParticipantUserId(enrollee.getParticipantUserId())
+                        .createdAt(Instant.now())
+                        .lastUpdatedAt(Instant.now())
+                        .status(ImportItemStatus.FAILED)
+                        .message(e.getMessage())
+                        .detail(Arrays.toString(e.getStackTrace())).build();
+            }
+
+            ImportItem importItem1 = importItemService.create(importItem);
```

Comment:
```suggestion
           importItem = importItemService.create(importItem);
```
That will prevent someone accidentally referencing the stale unsaved importItem after it's been saved